.github/workflows/pr-opened.yml

```diff
@@ -15,6 +19,8 @@ jobs:
     steps:
       - uses: actions/github-script@98814c53be79b1d30f795b907e553d8679345975 # ratchet:actions/github-script@v6
         with:
+          # no idea how to break this long line
+          # yamllint disable rule:line-length
```

Comment:
Normally this is possible with | or > like. > should keep the line breaks, > replace them by spaces

```yaml
abc: >
   some text
   some more